### ISSUE TITLE
2 blank lines in body pane but 1 blank line in @file

### ISSUE THREAD

**Issue by wangzhaohe** [leo-editor/leo-editor#1037]
There are 2 blank lines in body pane but 1 blank line in @file after save to local file.

at.putBody now regularizing trailing whitespace more simply.  This is made possible because Leo no longer generates `@nonl` sentinels.

The same code fixes both #1048 and #1037.

**Comment by edreamleo** [comment]
I am going to reopen this issue, and change it to a bug.  While experimenting with`@clean` I noticed annoying extra blank lines in nodes.

**Comment by edreamleo** [comment]
Fixed in the "output" branch. It will be merged only after more testing.

**Comment by edreamleo** [comment]
The "output" branch has been fully merged into "devel".
